packages/core/src/components/Tooltip/index.tsx

```diff
@@ -250,22 +308,37 @@ export class Tooltip extends React.Component<TooltipProps & WithStylesProps, Too
     const distance = unit / 2;
     const invert = inverted || Tooltip.inverted;
 
-    return (
-      <Overlay noBackground open={open} onClose={this.handleClose}>
+    const handleMouseEnter = popover ? this.handlePopupMouseEnter : undefined;
+    const handleMouseLeave = popover ? this.handlePopupMouseLeave : undefined;
+
+    const popupContent = (
+      <div
+        ref={this.handleTooltipRef}
+        role="tooltip"
+        className={cx(styles.tooltip, above ? styles.tooltip_above : styles.tooltip_below, {
+          width,
+          marginLeft: marginLeft[align as keyof StyleStruct],
+          marginTop: above ? -(tooltipHeight + targetRect.height + distance) : distance,
+          textAlign: align,
+        })}
+      >
         <div
-          ref={this.handleTooltipRef}
-          role="tooltip"
-          className={cx(styles.tooltip, above ? styles.tooltip_above : styles.tooltip_below, {
-            width,
-            marginLeft: marginLeft[align as keyof StyleStruct],
-            marginTop: above ? -(tooltipHeight + targetRect.height + distance) : distance,
-            textAlign: align,
-          })}
+          className={cx(styles.content, invert && styles.content_inverted)}
+          onMouseEnter={handleMouseEnter}
+          onMouseLeave={handleMouseLeave}
         >
-          <div className={cx(styles.content, invert && styles.content_inverted)}>
-            <Text inverted={invert}>{content}</Text>
-          </div>
+          <Text inverted={invert}>{content}</Text>
         </div>
+      </div>
+    );
+
+    if (popover) {
+      return open && <div style={{ position: 'absolute', zIndex: 1 }}>{popupContent}</div>;
```

Comment:
~these should be in `styles` ideally, not inline~

Edit: It would be one more addition to this PR, but I think we should update `Overlay` to [optionally apply the `userSelect: 'none'`](https://github.com/airbnb/lunar/blob/master/packages/core/src/components/Overlay/Portal/styles.ts#L13) style with a prop. 
Reasons:
- Overlay handles a lot of logic, like updating the position upon scroll (with complex scroll listeners), etc. which I'm not sure we handle here (?)
- This would enable us to use `onClose` irrespective of whether `popover=true/false`

I played with it in the storybook, and it doesn't seem like it's necessary. But I think better to add it as a prop to enable it so it's not a breaking change.

Reply:
if we don't use `Overlay`, we should move these styles into the stylesheet.